server/protocols/jmap/pom.xml

```diff
@@ -234,6 +234,11 @@
                     <artifactId>james-server-util-java8</artifactId>
                 </dependency>
                 <dependency>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-lang3</artifactId>
+                    <version>3.3.2</version>
+                </dependency>
+                <dependency>
```

Comment:
This fixup should not be a fixup of sorting of dependencies, but a fixup of the commit when you create the MessagePreview(Generator)